Tasks/DockerV1/task.json

```diff
@@ -268,14 +268,14 @@
                 "always": "Always",
                 "unlessStopped": "Unless stopped"
             },
-            "visibleRule": "command = Run an image && runInBackground = true",
```

Comment:
why remove this?

Reply:
command = Run an image || command = run  && runInBackground = true we can't do this rule as rule contains && and ||. And runInBackground  visible only 'Run an image' or 'run' set. so i removed this.